Kitodo-DataManagement/src/main/resources/db/migration/V2_29__Split_processes_and_templates.sql

```diff
@@ -0,0 +1,116 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+--
+-- Migration: Split processes and templates
+
+-- 1. Add template table
+--
+
+CREATE TABLE template (
+  id INT(11) NOT NULL AUTO_INCREMENT,
+  title VARCHAR (255) DEFAULT NULL,
+  outputName VARCHAR (255) DEFAULT NULL,
+  creationDate datetime DEFAULT NULL,
+  inChoiceListShown tinyint(1) DEFAULT NULL,
+  sortHelperStatus VARCHAR(255) DEFAULT NULL,
+  wikiField longtext,
+  project_id INT(11) DEFAULT NULL,
+  ruleset_id INT(11) DEFAULT NULL,
+  docket_id INT(11) DEFAULT NULL,
+  indexAction VARCHAR(6),
+  old_id INT(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) DEFAULT CHARSET=utf8mb4;
```

Comment:
Missing collate information like in other created tables like `COLLATE utf8mb4_unicode_ci`.